tests/system/src/test/java/test/robot/scenegraph/JDK8130122Test.java

```diff
@@ -84,6 +84,7 @@ public void testEmptyShapes() {
             testScene.setCamera(new PerspectiveCamera());
             testScene.setFill(Color.WHITE);
             testStage.setScene(testScene);
+            testStage.setAlwaysOnTop(true);
```

Comment:
This is redundant. `getStage()` already sets `alwaysOnTop`. If the window currently isn't always on top then we have a bigger problem.

Reply:
I see that thanks, i didn't notice it. I will remove the change.

I took https://github.com/openjdk/jfx/commit/56f2e17f#diff-43fd8d27727f352ddb3526031bd32e5d30aefa2bad722d1f3d21c4a0ca8a5a1f as an example and made this change.